src/ApparentHorizons/SpherePackIterator.hpp

```diff
@@ -0,0 +1,106 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///
+/// \file
+/// Defines class SpherePackIterator.
+
+//
+// iterate through an array with the layout of the Ylm-spectral
+// coefficients of SpherePack.
+//================================================================
+#pragma once
+
+#include <ostream>
+#include <vector>
+
+/// Iterates through the coefficients of a SpherePack spectral
```

Comment:
/// \ingroup Spectral

You will need to define the group in docs/GroupDefs.hpp

Reply:
SPHEREPACK

Reply:
Make the first sentence part of
/// \brief  First Sentence here.
Move the next sentence to the detail